Source/Core/DolphinWX/Config/WiiConfigPane.cpp

```diff
@@ -11,10 +11,11 @@
 #include "Core/ConfigManager.h"
 #include "Core/Core.h"
 #include "Core/IPC_HLE/WII_IPC_HLE.h"
-#include "DiscIO/Volume.h"
```

Comment:
This should be included because `OnSystemLanguageChoiceChanged` uses `DiscIO::IVolume::ELanguage`.